doc/papers/joss/paper.md

```diff
@@ -37,9 +37,10 @@ authors:
   - name: David Medina
     affiliation: 4
   - name: Will Pazner
-    affiliation: 2
     orcid: 0000-0003-4885-2934
-  - name: Thilina Rathnayake
```

Comment:
Wow! We had a couple of names misspelled! I'm sorry! Considering that I had taken them from the technical reports... ^_^''

Reply:
Not a mistake per se; he changed the way he anglicized the name and we've been slow to update consistently.